app/(components)/footer/footer.module.css

```diff
@@ -0,0 +1,11 @@
+.footer {
+  display: flex;
```

Comment:
Some of these styles might not be needed if we see what is available with the [Grid system](https://chakra-ui.com/docs/components/grid) in Chakra UI. If you get a chance to optimize the layout using the grid, let's do that but not too concerned about it for this PR. We can open a new task and do this as a follow up. You take the call.